yt_dlp/extractor/youtube.py

```diff
@@ -3739,6 +3758,22 @@ def process_manifest_format(f, proto, itag):
                             f['is_from_start'] = True
 
                         yield f
+
+                        # Fallback format url
+                        # Note: YouTube does not allow us changing the playback_host in the manifest_url.
+                        # So this will not work for external downloaders such as ffmpeg.
+                        fallback_url = self._build_fallback_playback_url(
```

Comment:
I'm thinking of removing the DASH fallback formats due to the inconsistency with the manifest url.